src/devices/Ad7193/samples/README.md

```diff
@@ -0,0 +1,140 @@
+private static int samplesTaken = 0;
+
+
+static void Main(string[] args)
+{
+	// set SPI bus ID: 0
+	// AD7193 CS Pin: 1
+	SpiConnectionSettings settings = new SpiConnectionSettings(0, 1);
+	settings.ClockFrequency = Ad7193.MaximumSpiFrequency;
+	settings.Mode = SpiMode.Mode3;
+	SpiDevice ad7193SpiDevice = SpiDevice.Create(settings);
+
+
+	ad7193 = new Ad7193(ad7193SpiDevice);
+	ad7193.AdcValueReceived += Ad7193_AdcValueReceived;
+
+	Console.WriteLine($"-- Resetting and calibrating AD7193.");
+	ad7193.Reset();
+	ad7193.PGAGain = Ad7193.Gain.X1;
+	ad7193.Averaging = Ad7193.AveragingModes.Off;
+	ad7193.AnalogInputMode = Ad7193.AnalogInputModes.EightPseudoDifferentialAnalogInputs;
+	ad7193.AppendStatusRegisterToData = true;
+	ad7193.JitterCorrection = true;
+	ad7193.Filter = 0;
+
+	Console.WriteLine($"AD7193 before calibration: offset={ad7193.Offset.ToString("x")}, full-scale={ad7193.FullScale.ToString("x")}");
+	ad7193.Calibrate();
+	Console.WriteLine($"AD7193  after calibration: offset={ad7193.Offset.ToString("x")}, full-scale={ad7193.FullScale.ToString("x")}");
+
+
+	Console.WriteLine("Starting 100 single conversions on CH0...");
+	ad7193.ActiveChannels = Ad7193.Channel.CH00;
+
+	for (int i = 0; i < 100; i++)
+	{
+		ad7193.StartSingleConversion();
+		ad7193.WaitForADC();
+		ad7193.ReadADCValue();
+		Thread.Sleep(25);
+	}
+
+	Thread.Sleep(1000);
+
+
+	Console.WriteLine();
+	Console.WriteLine();
+	Console.WriteLine("Starting continuous conversion on CH0 and CH1...");
+	ad7193.ActiveChannels = Ad7193.Channel.CH00 | Ad7193.Channel.CH01;
+	ad7193.StartContinuousConversion();
+
+	int loopcounter = 0;
+	while (true)
+	{
+		loopcounter++;
+		if (ad7193.HasErrors || (loopcounter % 50 == 0))
+		{
+			Console.WriteLine();
+			Console.WriteLine($"AD7193 status: {ad7193.Status}");
+			Console.WriteLine($"AD7193 mode: {ad7193.Mode}");
+			Console.WriteLine($"AD7193 config: {ad7193.Config}");
+			Console.WriteLine();
+			Thread.Sleep(1500);
+		}
+		Thread.Sleep(250);
+	}
+}
+
+private static void Ad7193_AdcValueReceived(object sender, Iot.Device.Ad7193.AdcValueReceivedEventArgs e)
+{
+	if (firstDataRead == DateTime.MinValue) firstDataRead = DateTime.UtcNow;
+	double secondsElapsed = (DateTime.UtcNow - firstDataRead).TotalSeconds;
+
+	samplesTaken++;
+
+	// show the results in every 0.25 seconds
+	if (secondsElapsed - lastChecked > 0.25)
+	{
+		double sps = (double)(samplesTaken - lastCount) / (secondsElapsed - lastChecked);
+
+		lastCount = samplesTaken;
+		lastChecked = secondsElapsed;
+
+		Iot.Device.Ad7193.AdcValue adcValue = e.AdcValue;
+
+		Console.WriteLine($"Channel {adcValue.Channel.ToString().PadLeft(2)}: {adcValue.Voltage.ToString("0.0000").PadLeft(11)} V | {adcValue.Raw.ToString("N0").PadLeft(13)} | {sps.ToString("N1").PadLeft(9)} SPS");
+	}
+}
+```
+
+## Result
+You can see that the values are oscillating between 0.0 V and 1.0 V at the rate of 4800 SPS, as expected.
+
+![Results in a PowerShell window](PowerShell_RaspberryPi_Output.png)
```

Comment:
cannot put comment there:
- please remove the checked in DLL
- please make sure all pictures are less than 100kB (i.e., convert to JPEG, lower quality of JPEG)

Reply:
I reduced the JPG file sizes, but the DLL is crucial for the deployment script to work, so I left it there.

Reply:
@andrasfuchs:
- we cannot have dlls in the repo, this is because no one can easily verify fidelity and also adds lots of risk (i.e. what if your machine is infected and the dll is also inadvertently infected - everyone using the script will be infected)
- all scripts should not be in the device samples - you should put them in the documentation with instructions and you can provide a link of where to get the dll not the dll itself, we should not do that per binding and rather globally since those won't differ per binding